chapters.md

```diff
@@ -3,31 +3,82 @@
 
 # Chapters
```

Comment:
The title of this patch `Start new Chapters` is not very specific nor does it explain what it adds and why.